src/js/program/program-controller.js

```diff
@@ -229,18 +234,19 @@ class ProgramController extends Eventable {
      * @returns {void}
      */
     backgroundActiveMedia() {
-        const { backgroundMedia, mediaController } = this;
+        this.adPlaying = true;
```

Comment:
Doesn't this.adPlaying === mediaController.background ?

Reply:
mediaController.background is set too late. This flag needs to be set first thing